packages/arb-token-bridge-ui/src/components/common/HeaderAccountPopover.tsx

```diff
@@ -55,6 +60,15 @@ function isDeposit(tx: MergedTransaction) {
   return tx.direction === 'deposit' || tx.direction === 'deposit-l1'
 }
 
+async function tryLookupUD(address: string) {
+  const UDresolution = Resolution.fromEthersProvider({});
```

Comment:
Let's have the provider passed in and not rely on what the default provider in the library is, same as what we do with the ENS lookups:

```suggestion
async function tryLookupUD(provider: JsonRpcProvider, address: string) {
  const UDresolution = Resolution.fromEthersProvider(provider);
```

Reply:
We'll have to include a polygon mainnet provider for it because behind the scenes their sdk is calling these 2 apis when the param is undefined:
`https://eth-mainnet.alchemyapi.io/v2/GmQ8X1FHf-WDEry0BBSn0RgjVhjHkRmS`
`https://polygon-mainnet.g.alchemy.com/v2/iG-oHZ2FvjqC9D43O5q9sj62out5ubsy`

To specify providers, the call has to be:
```typescript
Resolution.fromEthersProvider({uns: {
    locations: {
      Layer1: {
        network: 'mainnet',
        provider: l1Provider
      },
      Layer2: {
        network: 'polygon-mainnet',
        provider: polygonProvider
      }
    }
  }});
```
https://github.com/unstoppabledomains/resolution/blob/62dd1ccfc58fc192a6432269c046042b295d622d/src/Resolution.ts#L387-L400

These are the networks they support for the `network` string:
https://github.com/unstoppabledomains/resolution/blob/62dd1ccfc58fc192a6432269c046042b295d622d/src/types/index.ts#L119-L125

Should I add polygon to the [rpcURLs](https://github.com/OffchainLabs/arb-token-bridge/blob/master/packages/arb-token-bridge-ui/src/util/networks.ts#L20)?

Reply:
As discussed on Slack, we'll pass in an empty object until UD updates their lib to support our use case.

Filed issue: https://github.com/unstoppabledomains/resolution/issues/229